docs/webhooks/09-V2-Overview.md

```diff
@@ -1147,6 +1136,7 @@ title: incident.assign
   ]
 }
 ```
+&nbsp;
+### Latest Version
 
-<!-- type: tab-end -->
-
+Please go to [V3 webhook](../webhooks/01-Overview.md)
```

Comment:
nit: This working is weird. Maybe "Please see V3 webhooks"?

Reply:
changed in the next PR.